futurex_openedx_extensions/helpers/caching.py

```diff
@@ -71,3 +74,36 @@ def wrapped(*args: Any, **kwargs: Any) -> Dict[str, Any]:
 
         return wrapped
     return decorator
+
+
+def invalidate_cache(cache_name: str) -> None:
```

Comment:
I suggest that we use `None` instead of `__all__`. Because it's less error prune. So, `invalidate_cache()` will invalidate everything, looks very clear to me. What do you think?
```suggestion
def invalidate_cache(cache_name: str = None) -> None:
```